powermock-core/build.gradle

```diff
@@ -1 +1,3 @@
-
+dependencies {
+    compile("com.musala.atmosphere:openbeans-jxpath:0.0.1")
```

Comment:
As I can see the library is not being developed since 2016. Adding it as dependecies will bring it to all users of PowerMock but fix is needed only for Android users. 

I think that better approach is to rewrite `ConfigurationMapper` in a way that compatable both OpenJDK(Oracle Java) and Google Android Java.

Reply:
Sounds like a perfectly valid approach. Let me know if you'd like me to review your PR for it.